src/tiledimage.js

```diff
@@ -989,6 +975,53 @@ function onTileLoad( tiledImage, tile, time, image ) {
     tiledImage._needsDraw = true;
```

Comment:
I think this needsDraw happens too soon now...it should go inside the completionCallBack of setTileLoaded, so it happens after the tile is actually truly loaded.